### DISCUSSION TITLE
my industry session shows incomplete

### DISCUSSION THREAD

**Discussion by ghost** [github/india#312] [Students]
i have watched the videos on industry session as per the condition not to skip and all in microsoft edge browser still it shows incomplete. how to complete the industry session.

**Comment by d4rkn1gh7** [comment]
> i have watched the videos on industry session as per the condition not to skip and all in microsoft edge browser still it shows incomplete. how to complete the industry session.

Similar issue with me as well, It is showing completed but not turning into green

**Comment by saaiaadityaayenduri1** [comment]
Should we watch the industry sessions only in online or can we watch them by downloading??
